src/serac/physics/utilities/functional/detail/hexahedron_L2.inl

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and

```

Comment:
Extremely minor: the H1 file is called `hexahedron_h1.inl`, so should the L2 file be called `hexahedron_l2.inl` for consistency in terms of capitalization? I realize the lowercase L is sort of ambiguous so maybe h1 -> H1 instead?